config/config.go

```diff
@@ -165,16 +168,40 @@ func GetConfig(configPath *string) (*Config, error) {
 		Database: rawCfg.Database,
 	}, nil
 }
-
 func readConfigFile(configPath *string) (*rawConfig, error) {
 	var rawCfg rawConfig
-	file, err := ioutil.ReadFile(*configPath)
+	// read main config file.
+	mainFileByte, err := ioutil.ReadFile(*configPath)
+	if err != nil {
+		return nil, errors.Wrap(err, "read config file")
+	}
+	if _, err := toml.Decode(string(mainFileByte), &rawCfg); err != nil {
+		return nil, errors.Wrap(err, "read config file")
+	}
+	// if no sub config file
+	if rawCfg.Include == "" {
+		return &rawCfg, nil
+	}
+	// read sub config files.
+	dir, err := ioutil.ReadDir(rawCfg.Include)
 	if err != nil {
-		// err
 		return nil, errors.Wrap(err, "read config file")
 	}
-	// json.Unmarshal(file, &rawCfg)
-	if _, err := toml.Decode(string(file), &rawCfg); err != nil {
+	// merge config.
+	buffer := make([][]byte, 1024)
```

Comment:
Why do you allocate 1024 empty slices here?

Reply:
sorry